text/0000-compact-slates.md

```diff
@@ -0,0 +1,500 @@
+    }
+  ]
+}
+```
+
+The 'return' slate from the recipient to the originator is expected to be larger, as it must include the recipient's outputs/inputs and proofs (as the party who posts the transaction needs all of this information). However, compacting the slate at each stage of its journey allows for future possibilities that may have been more difficult in previous iterations of the slate format. For instance, an initiator taking Grin payments might be expected to be better equipped to receive Grin transactions. They could present the payer with an invoice transaction for the amount compacted to the size of a QR code, which the payer could scan, accept, and then post back to the invoicer's listening wallet infrastructure.
+
+Compacting the slate also acts as a minor privacy-enhancer by hiding the initiator's outputs from the other party.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+### Slate Definition - JSON
+
+Entries prefixed with `//` denote fields that may be omitted, as well as their default assumed values. `#` denotes a comment for the purposes of illustration.
+
+Field ordering is canonical.
+
+```
+{
+# These fields are always present
+  "ver": "4:3",
+  "id": "mavTAjNm4NVztDwh4gdSrQ",
+  "sta": "S1",
+  "off": "0gKWSQAAAADTApZJAAAAANQClkkAAAAA1QKWSQAAAAA=",
+
+# These fields may or may not be present
+//"num_parts: 2,
+//"fee": "8000000",
+//"amt": "1000000000",
+//"feat": 0,
+//"ttl": null,
+
+# Sigs is always present with at least one entry
+  "sigs": [
+    {
+      "xs": "A5sKZUTyGG1FogzBtH8ZvGaVVOFFxusytVLN0rdpX2DE",
+//    "part": null,
+      "nonce": "A1MbUjB5g4fjZ3fTYATVBA8TADHGG77D8gHpzbQTl/IE"
+    }
+  ]
+
+# These structs may or may not be present
+
+//"coms": null,
+//"proof": null,
+//"feat_args": null
+}
+```
+
+A description of all fields and their meanings is as follows:
+
+### Top-Level Slate Struct
+
+##### Fields - Always present
+* `ver` - The slate version and supported block header version, separated by a `:`
+* `id` - The slate's UUID, standard hex-string encoding for UUIDs
```

Comment:
Isn't the `id` shortened with https://github.com/seigert/shorter-uuid-rs and Base64 Encoded? Maybe worth mentioning the reference somewhere (might have missed it).

Reply:
No, I reverted back to standard UUiD format because it made it too difficult to reconcile slates with transaction log entries. The space savings isn't as significant now that there's a binary representation.

Reply:
I'm seeing this format in several places:
```
   "id": "mavTAjNm4NVztDwh4gdSrQ",
```
Am I missing something?